adobe/spectrum-web-components
# fix(split-view): expand accessible attribute usage and HCM delivery

## Description
- use latest Spectrum CSS dependency to unlock HCM delivery
- add `aria-controls` management
  - this is a little shaky, for the shadow DOM boundary, but I've added the ID internally and externally
    - internally for being able to connect with current browser APIs
    - externally for what should _really_ be controlled when under development APIs land
- add `aria-orientation`
- add default `aria-label`

## Related issue(s)
- fixes #3381
- fixes #3384
- fixes #3386
- refs #3390

## How has this been tested?
    1. Go [here](https://split-view--spectrum-web-components.netlify.app/storybook/index.html?path=/story/split-view--horizontal-resizable)
    2. Inspect the DOM for the updates
    1. Visit the HCM VRTs below
    2. See that the splitter is now visible in that context

## Screenshots (if appropriate)
<img width="721" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/b8003508-e005-4e69-a995-cacb8d50e4a5">

## Types of changes

## Checklist
